src/assets/stylesheets/shared/_overrides.less

```diff
@@ -23,7 +23,7 @@
 .blank-slate-pf {
   /* stylelint-disable */
```

Comment:
Remove `/* stylelint-disable */` comment call outs that surround this property, since you remove the `!important`.